aws/templates/id/id_rds.ftl

```diff
@@ -47,10 +47,14 @@
                     {
                         "Name" : "MasterUserName",
                         "Default" : "root"
-                    }
+                    },
                     {
                         "Name" : "CharacterLength",
                         "Default" : 20
+                    },
+                    {
+                        "Name" : "VariablePrefix",
```

Comment:
Can we make this a bit more specific? It only affects encrypted material so maybe 

"EncryptedVariablePrefix"  ?